nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -58,6 +59,15 @@
 
     public static final String COMMENT_KEY = "comment";
 
+    /**
+     * Define whether virtual user (non-existent user) is allowed in Nuxeo automation. If allowed, Nuxeo server will
+     * not check the user existence during automation execution. Set this property to true if you use Nuxeo computed
+     * user or computed group.
+     *
+     * @see <a href="https://jira.nuxeo.com/browse/NXP-21889">NXP-21889</a>
```

Comment:
It's wasn't in the  8.10 release so it shouldn't be `@since 8.10`. We usually don't update the `@since` for backports, if someone looking at the 8.10 maintenance branch sees a `@since 9.1` they'll know it's a backport anyway. If you want you can set `@since 8.10-HF04, 9.1` but it's usually overkill. Especially since new APIs shouldn't be introduced in backports, except in rare cases.